kratos/python_scripts/auxiliar_methods_adaptative_remeshing.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics
+
+def AdaptativeRemeshingRunSolutionLoop(analysis):
+    """This function executes the solution loop of the AnalysisStage for cases where remeshing may be considered
+
+        Keyword arguments:
+        analysis The AnalysisStage to be reinitialized
+    """
+
+    # If we remesh using a process
+    computing_model_part = analysis._GetSolver().GetComputingModelPart()
+    root_model_part = computing_model_part.GetRootModelPart()
+
+    while analysis.time < analysis.end_time:
```

Comment:
```suggestion
    while self.KeepAdvancingSolutionLoop():
```

Reply:
What is this?, is this method implemented?

Reply:
In fact it is analysis.KeepAdvancingSolutionLoop():